lib/components/src/layout/desktop.js

```diff
@@ -2,15 +2,64 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { localStorage, window } from 'global';
 
+import { injectGlobal } from 'emotion';
 import styled from 'react-emotion';
 import throttle from 'lodash.throttle';
 
 import SplitPane from 'react-split-pane';
 
-import USplit from './usplit';
 import Dimensions from './dimensions';
 
+// eslint-disable-next-line no-unused-expressions
```

Comment:
🤦‍♂️

Reply:
Well it makes sense, it's a deliberate side-effect..